W/wxWidgets/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+using BinaryBuilder, Pkg
+
+name = "wxWidgets"
+version = v"3.1.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/wxWidgets/wxWidgets/releases/download/v$(version)/wxWidgets-$(version).tar.bz2",
+                  "3ca3a19a14b407d0cdda507a7930c2e84ae1c8e74f946e0144d2fa7d881f1a94")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/wxWidgets-*/
+apk add g++ linux-headers samurai
```

Comment:
Why?

Reply:
I was guessing it might help with musl linux....
I now know it wont...😑

Reply:
You don't have to do anything for musl-based systems: #2347 would do, if only `apk` wasn't broken again

Reply:
Hey, thanks for telling me!
The latest commit fixed freebsd! Wohoo!
My only concern now is apple I guess....

Reply:
> You don't have to do anything for musl-based systems: #2347 would do, if only apk wasn't broken again

I don't think so....

Reply:
The error is completely different now, GTK can be found without issues.  _**Now**_ you can do something for this platforms